build.gradle

```diff
@@ -73,7 +73,7 @@ allprojects {
 	dependencies {
 		minecraft "com.mojang:minecraft:$Globals.mcVersion"
 		mappings "net.fabricmc:yarn:${Globals.mcVersion}${Globals.yarnVersion}:v2"
-		modCompile "net.fabricmc:fabric-loader:0.8.9+build.203"
+		modCompile "net.fabricmc:fabric-loader:0.9.2+build.206"
```

Comment:
In my opinion, it might be good to require 0.9.2 in fabric.mod.json

Reply:
For registry sync's json file specifically.

Reply:
I mean, *technically* it doesn't require it.
But all mods calling it from their ModInitializer require it.
I'd be happy to change it from... uhh... >=0.4.0 to >= 0.9.2.